stable/kommander-karma/README.md

```diff
@@ -32,14 +32,14 @@ karma:
       traefik.ingress.kubernetes.io/auth-type: "forward"
       traefik.ingress.kubernetes.io/auth-url: "http://traefik-forward-auth-kubeaddons.kubeaddons.svc.cluster.local:4181/"
       traefik.ingress.kubernetes.io/priority: "2"
-    path: "/kommander/monitoring/karma"
+    path: "/ops/portal/kommander/monitoring/karma/"
```

Comment:
ugh, just caught that this slash shouldn't have been added. putting up a fix asap.